Makefile

```diff
@@ -38,10 +38,17 @@ CPPFLAGS = -Wall -I$(SRC)include -std=$(STD) -pthread \
 ARFLAGS = rcs
 LDFLAGS = -pthread
 LDLIBS = $(TEST_LIBS)
+LN = ln
+CP = cp
+MKDIR = mkdir
 
 SUDO =
 
 CCTZ_LIB = libcctz.a
+CCTZ_STATIC_LIB = $(CCTZ_LIB)
+CCTZ_SHARED_LIB = libcctz.so.2.0
+CCTZ_SHARED_SONAME = $(basename $(CCTZ_SHARED_LIB))
```

Comment:
It removes the last part of the version number, so it becomes `libcctz.so.2` (drops the `.0`)